packages/graphback-cli/src/utils/generateHelper.ts

```diff
@@ -0,0 +1,71 @@
+import chalk from 'chalk';
+import { accessSync, readFileSync, writeFileSync } from 'fs';
+import { GlobSync } from 'glob'
+import { GraphQLBackendCreator, IGraphQLBackend, KnexResolverManager, PostgresSchemaManager } from 'graphback-core'
+import { logError, logInfo } from '.';
+
+/**
+ * Message after command execution
+ */
+function postCommandMessage(): void {
+  logInfo(`
+Successfully generated schema and resolvers :tada:.  
+
+Next steps:
+1. Run ${chalk.cyan(`graphback db`)} to create postgres database.
+2. Run ${chalk.cyan(`docker-compose up -d`)} followed by ${chalk.cyan(`npm start`)}
+`)
+}
+
+/**
+ * Generate schema and resolvers using graphback-core and
+ * write them into generated folder
+ */
+async function generateBackend(): Promise<void> {
+  try{
+    accessSync(`${process.cwd()}/model`)
+    const model = new GlobSync('model/*.graphql', { cwd: process.cwd()})
+    
+    if(model.found.length === 0) {
+      logError(`No graphql file found inside model.`)
+      process.exit(0)
+    }
+
+    const path = `${process.cwd()}/${model.found[0]}`
```

Comment:
Why only single file. Why we cannot load more than one and just concatenate they are types so this should be nice. We should promote more than one file. Our examples could be one file but devs may have 50 to 150 types inside their schema. It is not feasible to have it in single file

Reply:
Hadn't thought about this, but this is a good idea. We should do it.